test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_ontimeinterval_expiration.test

```diff
@@ -30,75 +30,91 @@ accumulatorStart
 
 --SHELL--
 
-url="/v1/updateContext"
```

Comment:
Is there any real change in this .test or just cleaning? (In theory, the cache funcationality shouldn't affect to existing functional tests).